resource_estimator/src/estimates/modeling/physical_qubit.rs

```diff
@@ -13,6 +13,64 @@ use super::super::{
 };
 use serde::{de::Error, Deserialize, Serialize};
 
+pub trait TPhysicalQubit {
```

Comment:
T is for trait. Correct?
Do we already follow this pattern?

Reply:
No, this is not final. A good name would be `PhysicalQubit`, but we are already using it. I still want to change the names.

Reply:
In fact, we don't need a trait on the physical qubits, so I removed this one. We require no properties on the physical qubit for estimation, as it is all derived from the ErrorCorrection trait.